tests/conftest.py

```diff
@@ -1667,9 +1667,17 @@ def __dut_reload(duts_data, node=None, results=None):
         logger.error('Missing kwarg "node" or "results"')
         return
     logger.info("dut reload called on {}".format(node.hostname))
-    node.copy(content=json.dumps(duts_data[node.hostname]["pre_running_config"], indent=4),
+    node.copy(content=json.dumps(duts_data[node.hostname]["pre_running_config"]['host'], indent=4),
                  dest='/etc/sonic/config_db.json', verbose=False)
-    config_reload(node)
+    if node.is_multi_asic:
+        for asic_index in range(0, node.facts.get('num_asic')):
+            asic_ns = 'asic{}'.format(asic_index)
+            asic_cfg_file = "/tmp/{}_config_db{}.json".format(node.hostname, asic_index)
+            with open(asic_cfg_file, "w") as outfile:
+                outfile.write(json.dumps(duts_data[node.hostname]['pre_running_config'][asic_ns], indent=4))
+            node.copy(src=asic_cfg_file, dest='/etc/sonic/config_db{}.json'.format(asic_index), verbose=False)
+
+    config_reload(node, safe_reload=True, check_intf_up_ports=True)
```

Comment:
are safe_reload and check_intf_up required here?
If sanity is enabled, then critical process and interfaces status should be checked as a part of sanity.
Checking it here will add additional time.

Reply:
Was not aware that sanity check also polls/retries. Have removed the safe_reload and check_intf_up_ports with the latest commit.